compiler/compile/OMRCompilation.cpp

```diff
@@ -897,7 +897,7 @@ OMR::Compilation::getProfilingMode()
    if (!self()->isProfilingCompilation())
       return DisabledProfiling;
 
-   if (self()->getOption(TR_EnableJProfiling) || self()->getOption(TR_EnableJProfilingInProfilingCompilations))
+   if (self()->getOption(TR_EnableJProfiling) || !self()->getOption(TR_DisableJProfilingInProfilingCompilations))
```

Comment:
For the review : We have two options used to enable JProfiling compilation. First one is used for Warm bodies while second one is used for Profiling compilations. We will clean up how we are enabling/disabling JProfiling once ongoing work for JProfiling in Warm Bodies gets completed.

Reply:
So the ```TR_EnableJProfiling``` option is still there even though JProfiling is already enabled at some opt levels ? Can that option be renamed to make it clear that it applies only to warm bodies ?

Reply:
I agree that we need to rename the `TR_EnableJProfiling` to something that makes more sense as it will be used for the method bodies with patchable profiling `(<= warm bodies, can use something like `TR_EnablePatchableJProfiling)`.
Renaming `TR_EnableJProfiling` would need to change lot of places in `openj9` as well which creates build dependency. I am thinking to do it as separate PR and have JProfiling for Profiling compilations enabled first so that it can go into testing cycles.

Reply:
Okay I would like to see that separate PR soon so that it's not forgotten. Do we need an issue for it ?

Reply:
Yes, just opened up the issue : https://github.com/eclipse/omr/issues/4265